book/team.yaml

```diff
@@ -354,3 +354,26 @@ people:
         link: https://github.com/YoungHyunKoo
     user_groups:
       - Tutorial Lead
+  # =============
+  - title: Jullian Williams
+    avatar: https://avatars.githubusercontent.com/u/46701190?v=4
+    role: Doctoral research coordinator
+    organizations:
+      - name: The University of Texas at San Antonio
+    bio: > Jullian Williams is a recent PhD graduate from the University of Texas at San Antonio in Civil and Environmental Engineering. 
+    Jullian is a built environment project specialist, atmospheric scientist and teritary level teacher. She specializes in the use of geospatial software
```

Comment:
@jcbw  - The indendation needs to be adjusted for the 'bio' section. Everything into that needs to have two spaces
```YAML
bio: > Jullian Williams is a recent PhD graduate from the University of Texas at San Antonio in Civil and Environmental Engineering.
  Jullian is a built environment project specialist, atmospheric scientist and teritary level teacher. She specializes in the use of geospatial software 
  and coding platforms to engineer geospatial science and solutions.
```